src/log_capture.jl

```diff
@@ -108,6 +114,11 @@ _has_logs(ts::TestSetup) = filesize(logpath(ts)) > 0
 # The path might not exist if a testsetup always throws an error and we don't get to actually
 # evaluate the test item.
 _has_logs(ti::TestItem, i=nothing) = (path = logpath(ti, i); (isfile(path) && filesize(path) > 0))
+_mem_watermark() = string(
```

Comment:
Please can you add some docs, maybe just as a comment on this function, for what `maxrss` is as i expect lots of people not to know what it is / why it's useful info (at least i didn't 😛)